integration/src/test/java/org/mule/test/core/context/notification/processors/MessageProcessorNotificationTestCase.java

```diff
@@ -335,9 +335,9 @@ public void collectionAggregator() throws Exception {
     specificationFactory = () -> new Node()
         .serial(pre()) // open Splitter, unpacks three messages
         .serial(prePost()) // 1st message on Logger
-        .serial(prePost()) // gets to Aggregator
+        .serial(pre()) // gets to Aggregator
```

Comment:
Discussed this with others and correct behaviour. If there are any issues then we can re-visit.